ledger-service/http-json/src/main/scala/com/digitalasset/http/Endpoints.scala

```diff
@@ -432,7 +436,7 @@ class Endpoints(
       metrics: Metrics,
   ): Future[Error \/ SearchResult[Error \/ JsValue]] = {
     for {
-      it <- EitherT.eitherT(inputAndJwtPayload[JwtPayload](req))
+      it <- inputAndJwtPayload[JwtPayload](req).leftMap(it => it: Error)
```

Comment:
I suggest this generally, for the purposes of finding it more easily for utility-ification later.

```suggestion
      it <- inputAndJwtPayload[JwtPayload](req).leftMap(identity[Error])
```

Reply:
Addressed by https://github.com/digital-asset/daml/pull/12075/commits/d5c4565527a4d8ee43c9ec2214d927eef4309981